scrapers/thumbnail_frame_matcher/thumbnail_frame_matcher.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+require "open3"
+require "json"
+require 'base64'
+require 'securerandom'
+require_relative "../rb_common/rb_common"
+
+class ThumbnailFrameMatcher
+  PTS_TIME_REGEX = /pts_time:([\d\.]+)/
+  TEMPORARY_DIRECTORY = "/generated/tmp/"
```

Comment:
For users not running the docker container this directory needs to be configured